app/models/manageiq/providers/google/cloud_manager/event_catcher/stream.rb

```diff
@@ -35,12 +43,22 @@ def events
         subscription.acknowledge(msgs)
       end
     end
+  rescue Fog::Errors::Error
```

Comment:
I haven't looked into this but this error it seems like it may include other exceptions besides unreachable.  Not a problem now but something to consider if it's a problem.